package.json

```diff
@@ -28,11 +28,15 @@
   },
   "homepage": "https://github.com/soswow/last-branches-cli",
   "scripts": {
-    "start": "node bin/last-branches.js"
+    "start": "node bin/last-branches.js",
+    "build": "tsup"
   },
   "dependencies": {
+  },
+  "devDependencies": {
```

Comment:
The dependencies need to be moved to dev so that tsup includes it in the compilation.

In dependencies, it assumes its available upon execution